README.md

```diff
@@ -45,10 +45,25 @@ topos --help
 
 Find more about how topos works in the [documentation](https://docs.topos.technology/).
 
+### Topos Docker image
```

Comment:
I'm not sure why this sub-section in the main README btw. This somewhat differs from the rest of the README that is mainly for node runners, otherwise we do have docker instruction in docs.topos.technology.

cc @wyhaines

Reply:
Should I remove it?